pkg/deploy/defaults.go

```diff
@@ -19,7 +19,7 @@ import (
 const (
 	defaultCheServerImageRepo           = "eclipse/che-server"
 	defaultCodeReadyServerImageRepo     = "registry.redhat.io/codeready-workspaces/server-rhel8"
-	defaultCheServerImageTag            = "7.1.0"
+	defaultCheServerImageTag            = "7.2.0"
```

Comment:
@davidfestal Can we have nightly defaults in master to have nightly Che deployed along with nightly che-operator?

Reply:
Not for now.
This was already discussed but would need a wider refactoring.
For now nightlies are managed differently.